core/src/test/java/org/infinispan/atomic/BaseAtomicMapLockingTest.java

```diff
@@ -188,24 +183,12 @@ protected void assertKeysLocked(int index, Object... keys) {
       }
    }
 
-   protected boolean hasKeyMappedTo(boolean toLockOwner, Collection<Object> keys) {
-      for (Object key : keys) {
-         boolean onLockOwner = DistributionTestHelper.isFirstOwner(cache(0), key);
-         if ((toLockOwner && onLockOwner) || (!toLockOwner && !onLockOwner)) {
-            return true;
-         }
-      }
-      return false;
-   }
-
-   private void resetBeforeMethod() {
+   @AfterMethod
```

Comment:
Needs `alwaysRun=true`, otherwise TestNG won't run it after a failed test. You could also use `@BeforeMethod` instead.